Makefile

```diff
@@ -250,11 +258,13 @@ controller-gen: $(CONTROLLER_GEN) ## Download controller-gen locally if necessar
 $(CONTROLLER_GEN): $(LOCALBIN)
 	test -s $(CONTROLLER_GEN) || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_GEN_VERSION)
 
-.PHONY: yq
-yq: $(YQ) ## Download yq locally if necessary.
+## Download yq locally if necessary.
 $(YQ): $(LOCALBIN)
```

Comment:
This is moved to a non-phony task, so that other non-phony tasks can depend on it. The old `yq` below just depends on it to act the same as before.

Reply:
Unrelated change, should not be part of this PR

Reply:
@ykaliuta IMHO it's related because this PR creates a non-phony rule, and it wanted a non-phony task to ensure existence $(YQ), so that was added also.

Is the part that you're saying is unrelated, that the existing yq rule was modified also? That could be reverted so that there are duplicate recipes in both the `.PHONY yq` and `$(YQ)` rules, but it seems like a good opportunity to share the recipe between both?

Reply:
I understand your intention and it sounds reasonable, but the actual change does not look as reflecting it. `yq` is phony and $(YQ) is not phony in both versions.

BTW, dependency on $(LOCALBIN) is not fully correct, it should be `| $(LOCALBIN)`. I remember we discussed it with Bartosz, he was talking about cleanup patch :), but did not submit it yet.